### PR TITLE
Adding Hoplix domains

### PR DESCRIPTION
**Pull request by W3b3r**
* [X] Description of Organization
* [X] Reason for PSL Inclusion
* [X] DNS verification via dig
* [X] Run Syntax Checker (make test)

* [X] Each domain listed in the PRIVATE section has and shall maintain at least two years remaining on registration.

Description of Organization
====

Organization Website: https://hoplix.com

I'm a Senior Software Engineer at Hoplix S.R.L., a company of print on demand product.

Reason for PSL Inclusion
====
Hoplxi S.R.L. offer free service of hosting on third level of hoplix.shop domain
We'd like to properly isolate each subdomain to prevent global cookie setting on hoplix.shop and our users need to verify our subdomains by yourself for facebook ads

All our users can purchase and/or connect their own domain name to their shops, if our user don't want to purchase and/or connect their own domain we need to improve cookie security and isolate each subdomain cookies for all *.hoplix.shop domains.

In addition if they use and/or not use own domain name we need to consider, the request between *.hoplix.shop domains samesite, and needed to leaving them open to CSRF.

DNS Verification via dig
=======

```
dig +short TXT _psl.hoplix.shop
"https://github.com/publicsuffix/list/pull/1282"
```

<!--
For each domain you'd like to add to the list please create
a DNS verification record pointing to your pull request.

For example, if you'd like to add example.com and example.net
you would need to provide the following verifications:

```

```
dig +short TXT _psl.example.net
"https://github.com/publicsuffix/list/pull/XXXX"
```

Note that XXXX is replaced with the number of your pull request.
-->

make test
=========
```
============================================================================
Testsuite summary for libpsl 0.21.0
============================================================================
# TOTAL: 5
# PASS:  5
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================

```
<!--
Please verify that you followed the correct syntax and nothing broke

git clone https://github.com/publicsuffix/list.git
cd list
make test

Simply let us know that you ran the test
-->

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -13231,6 +13231,10 @@ myshopblocks.com
 // Submitted by Alex Richter <alex.richter@shopify.com>
 myshopify.com
 
+// Hoplix : https://www.hoplix.com
```
Comment (sleevi) [inline public_suffix_list.dat:13234]: Please sort this according to the Guidelines - https://github.com/publicsuffix/list/wiki/Guidelines

Comment (dnsguru) [reply public_suffix_list.dat:13234]: Please also provide a more substantial rationale for the request

Comment (W3b3r) [reply public_suffix_list.dat]: Ok i sorted the lines and also i provided more substantial retionale for the request.
Thanks for advice

**Comment by dnsguru** [comment]
So, this is tied to #1277 ?

**Comment by dnsguru** [comment]
@W3b3r is this related to ticket #1277 - am confused about the provided rationale and what this request has to do with ticket.io - this looks like a copy/paste - can you please provide a rationale for YOUR project as basis for YOUR request?

**Comment by W3b3r** [comment]
> @W3b3r is this related to ticket #1277 - am confused about the provided rationale and what this request has to do with ticket.io - this looks like a copy/paste - can you please provide a rationale for YOUR project as basis for YOUR request?

Hi i updated the  substantial retionale for the request.

**Comment by dnsguru** [comment]
#1245

**Comment by dnsguru** [comment]
Closed ; FB issue see #1245

**Comment by n8schloss** [comment]
Hi @W3b3r,

Facebook recently updated our [help center guidance](https://www.facebook.com/business/help/126789292407737?id=1205376682832142) around this and I'd like to specifically call out this section:

> Our current efforts are designed to support preexisting Public Suffix List domain registrations or eTLDs. This support is in line with Apple’s recent Private Click Measurement update. There are other technical implications if a domain is registered as a Public Suffix that a business should consider (for example, the domain that is registered on the Public Suffix List cannot have its own cookies) and as such, we do not recommend that clients register their domains on the Public Suffix List specifically for Facebook event configuration.

For hoplix.shop specifically it does indeed seem like you're hosting entirely separate businesses on each subdomain and that there isn't any functionality on the root hoplix.shop domain since your organizations main website is at hoplix.com. As long as cookie security a motivation here then I think this request seems legitimate.

**Comment by W3b3r** [comment]
Hi @n8schloss,

thanks a lot, btw seems that @dnsguru don't want to add hoplix.shop to public suffix

@dnsguru I need to create a new request for insert hoplix.shop on public suffix?

Thanks in advance
Kind regards

**Comment by dnsguru** [comment]
Please see the (acceptance/validation)[https://github.com/publicsuffix/list/wiki/Guidelines#validation-and-non-acceptance-factors] section of the Wiki

There is clearly stated guidance against use of the PSL as a workaround for third party limits.  If the objective was NOT to work around security / limits of third parties, this PR might be able to proceed, but it is.

That said, as you see N8 and others are here from FB and we are in open discussion about the overall matter, which at the time of your PR was wontfix/close.   We can unclose the PR, so you do not need to re-submit, but it will still be merged and will remain labelled as wontfix until there is a solve for the general issue of attempting to leverage the PSL to workaround limits. 

That condition is there because the browsers and other orgs that opt to voluntarily use the PSL do so because it is not used to bypass their stuff.  Once we'd open the floodgates on the matter, it might have the effect of voluntary use of the PSL withdrawn by browsers or others.  We don't want to pull that loose thread on this sweater.

**Comment by dnsguru** [comment]
Re-Opened, but remains wontfix until larger matter of attempting to leverage PSL to work around third party limits is addressed.

**Comment by W3b3r** [comment]
@dnsguru 
Thanks for re-open my request

let me explain better the reson for PSL include:

all our users can purchase and/or connect their own domain name to their shops, if our user don't want to purchase and/or connect their own domain we need to improve cookie security and isolate each subdomain cookies for all *.hoplix.shop domains.

In addition if they use and/or not use own domain name we need to consider, the request between *.hoplix.shop domains samesite, and needed to leaving them open to CSRF.

Also each user for each subdomain can install thirdy party apps aviable on our app store.

This explanation can be considered relevant for acceptance/validation section of the Wiki?

**Comment by n8schloss** [comment]
Posting to let you know that Facebook has updated our guidance to include a validation flow run by Facebook staff. We encourage anyone requesting addition to the PSL as it relates to Facebook and iOS 14 interactions to make a request directly to Facebook via the form linked from our Help Center Article: https://www.facebook.com/business/help/126789292407737  

In the form, given that you have already raised a pull request here, please include a link to this pull request in the description.

We have agreed to do this to reduce the burden on PSL maintainers, ensure that you are asking to be on the PSL for the right reasons, and to allow us to collate and understand the issues that businesses are facing as part of this shift. 

After Facebook has assessed one of these requests we will make a recommendation or no recommendation decision, the PSL maintainers will make the final determination and we will coordinate with them on next steps accordingly.

**Comment by dnsguru** [comment]
excellent, closing this PR - please reference this PR# should there be a new PR opened

**Pull request closed without merging**